cmd/fs-v1.go

```diff
@@ -226,6 +226,21 @@ func (fs *FSObjects) StorageInfo(ctx context.Context, _ bool) StorageInfo {
 	return storageInfo
 }
 
+// GetRWLock - returns a locked cluster-wide lock
+func (fs *FSObjects) GetRWLock(ctx context.Context, name string, timeout, minimum time.Duration) (RWLocker, error) {
```

Comment:
We can use NewNSLock instead of GetRWLock() its an overkill anyways @wlan0

Reply:
Yeah, i'll fix that everywhere